views/partials/_contact-panel.njk

```diff
@@ -1,6 +1,6 @@
 {%- set fileEditURL = "https://github.com/alphagov/govuk-design-system/edit/main/src/" + path + "/index.md.njk" -%}
 {% if section === 'Styles' or section === 'Components' or section === 'Patterns'%}
-  <h2 class="govuk-heading-l govuk-!-padding-top-7" id="help-improve-this-page">Help improve this page</h2>
+  <h2 class="govuk-heading-l govuk-!-padding-top-7" id="help-improve-this-page">Help improve this {% if section === 'Styles'%}style{% elif section === 'Components' %}component{% elif section === 'Patterns' %}pattern{%else%}page{% endif %}</h2>
```

Comment:
I reckon you don't need the big `if`. You can instead spit out the `section` string, which we already know is one of the 3 we want from line 2, with some nunjucks formatting tricks:

```suggestion
  <h2 class="govuk-heading-l govuk-!-padding-top-7" id="help-improve-this-page">Help improve this {{ section | lower }}</h2>
```

Reply:
It's a neat suggestion, but unfortunately the `section` is plural…

<img width="759" alt="Screenshot 2022-05-25 at 09 12 46" src="https://user-images.githubusercontent.com/121939/170214354-7fc29b97-ae78-4392-a92d-651adb10db05.png">

Reply:
Oh no 😂

Reply:
I won't be bested that easily:

```nunjucks
Help improve this {{ section.slice(0, -1) | lower }}
```